src/raiden_libs/cli.py

```diff
@@ -132,10 +133,11 @@ def blockchain_options(contracts: List[str]) -> Callable:
     ]
 
     arg_for_contract = {
-        CONTRACT_TOKEN_NETWORK_REGISTRY: "token-network-registry",
+        CONTRACT_TOKEN_NETWORK_REGISTRY: "token-network-registry-contract",
```

Comment:
This needs to be updated in the docs.

Reply:
Do you think the change is worthwhile or should I undo it? It just didn't seem consistent

Reply:
When I added it, I intentionally kept the name to keep it the same as for the client. But now neither have to be entered explicitly, usually. I don't mind either way.

Reply:
Let's do it in a different PR